src/Model/DatagridManagerInterface.php

```diff
@@ -16,6 +16,8 @@
 /**
  * A datagrid manager is a bridge between the model classes and the admin datagrid functionality.
  *
+ * NEXT_MAJOR: Remove this interface
```

Comment:
Can you please add also a comment in https://github.com/sonata-project/SonataAdminBundle/blob/fe00c88dbad4a4345332cb6a237a29c53116bd95/src/Model/ModelManagerInterface.php#L27.

Reply:
Done